crates/wasmtime/src/trap.rs

```diff
@@ -264,6 +264,11 @@ impl Trap {
         }
     }
 
+    /// Displays the error reason for this trap.
```

Comment:
Could this elaborate a bit on what the method does? For example I think this should explain how this is different from `Display for Trap` and an example situation you might end up using this.